### ISSUE TITLE
profile image not updating

### ISSUE THREAD

**Issue by NitinNR** [chatwoot/chatwoot#3132]
using self hosted version.

is there any need to configure the env variables ?

**Comment by pranavrajs** [comment]
@NitinNR Please provide more context and follow the issue template. Please add logs from your instance, screenshots/screenshots without which debugging is not possible.

**Comment by tejaswinichile** [comment]
https://user-images.githubusercontent.com/10827585/150129000-0d321d74-0d42-45ba-ad15-9163f0628357.mov

This is working fine for me.

@NitinNR Please reopen it if you are still facing the issue.

**Comment by vedangdate** [comment]
I am putting up the screen recording for a reference.
PS: I recorded only the browser so the explorer window from which the file is being uploaded cant be seen, but it does open so I didn't make another recording.

[The Screen Recording](https://1drv.ms/v/s!Av0Qffzgfa4fgfRIY8xuoFWnaeQDzg)

**Comment by tejaswinichile** [comment]
@vedangdate If this is a self-hosted version can you check the logs and add it here? Which version of chatwoot are you using?

**Comment by vedangdate** [comment]
Yes, it is a self hosted version using CapRover on a LightSail Ubuntu instance. 
I started using Chatwoot yesterday, so I am hoping the one I installed, using the docs, is the latest one.

Docker Image: 2.0.0

The logs are as follows:

2022-01-20T07:14:54.542966076Z I, [2022-01-20T07:14:54.542861 #1]  INFO -- : source=rack-timeout id=7bf69eb9-ef26-4dd0-9dad-4895e5d6d507 timeout=15000ms service=10ms state=completed
2022-01-20T07:14:54.563810080Z I, [2022-01-20T07:14:54.563633 #1]  INFO -- : Finished "/cable/" [WebSocket] for 49.36.59.244 at 2022-01-20 07:14:54 +0000
2022-01-20T07:14:54.564004008Z I, [2022-01-20T07:14:54.563881 #1]  INFO -- : RoomChannel stopped streaming from ArzPczNxJyZSkTPGeeUDt7WG
2022-01-20T07:14:55.069076227Z I, [2022-01-20T07:14:55.068890 #1]  INFO -- : source=rack-timeout id=efebfd46-48b4-499b-808b-97f9536c84d3 timeout=15000ms state=ready
2022-01-20T07:14:55.069725223Z I, [2022-01-20T07:14:55.069573 #1]  INFO -- : source=rack-timeout id=9b69831f-aac8-4d53-99f7-8063294233a8 timeout=15000ms state=ready
2022-01-20T07:14:55.071045351Z I, [2022-01-20T07:14:55.070914 #1]  INFO -- : [efebfd46-48b4-499b-808b-97f9536c84d3] Started GET "/api/v1/accounts/2/labels" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.071761429Z I, [2022-01-20T07:14:55.071644 #1]  INFO -- : source=rack-timeout id=cd066f94-1299-4ace-8953-5ebc9e100f75 timeout=15000ms state=ready
2022-01-20T07:14:55.074025295Z I, [2022-01-20T07:14:55.073890 #1]  INFO -- : source=rack-timeout id=39edc88f-ee04-4a15-b4f1-6ff8d9683e34 timeout=15000ms state=ready
2022-01-20T07:14:55.075601964Z I, [2022-01-20T07:14:55.075467 #1]  INFO -- : [cd066f94-1299-4ace-8953-5ebc9e100f75] Started GET "/api/v1/accounts/2/teams" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.076552986Z I, [2022-01-20T07:14:55.072643 #1]  INFO -- : source=rack-timeout id=e219039c-5bac-46f4-ac79-36686d262c57 timeout=15000ms state=ready
2022-01-20T07:14:55.077528665Z I, [2022-01-20T07:14:55.075090 #1]  INFO -- : [efebfd46-48b4-499b-808b-97f9536c84d3] Processing by Api::V1::Accounts::LabelsController#index as JSON
2022-01-20T07:14:55.077772434Z I, [2022-01-20T07:14:55.077630 #1]  INFO -- : [efebfd46-48b4-499b-808b-97f9536c84d3]   Parameters: {"account_id"=>"2"}
2022-01-20T07:14:55.078818123Z I, [2022-01-20T07:14:55.074177 #1]  INFO -- : [9b69831f-aac8-4d53-99f7-8063294233a8] Started GET "/api/v1/accounts/2/inboxes" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.079537323Z I, [2022-01-20T07:14:55.079388 #1]  INFO -- : [39edc88f-ee04-4a15-b4f1-6ff8d9683e34] Started GET "/api/v1/accounts/2/notifications/unread_count" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.082219386Z I, [2022-01-20T07:14:55.082064 #1]  INFO -- : [39edc88f-ee04-4a15-b4f1-6ff8d9683e34] Processing by Api::V1::Accounts::NotificationsController#unread_count as JSON
2022-01-20T07:14:55.082271916Z I, [2022-01-20T07:14:55.082186 #1]  INFO -- : [39edc88f-ee04-4a15-b4f1-6ff8d9683e34]   Parameters: {"account_id"=>"2"}
2022-01-20T07:14:55.084385678Z I, [2022-01-20T07:14:55.080964 #1]  INFO -- : [9b69831f-aac8-4d53-99f7-8063294233a8] Processing by Api::V1::Accounts::InboxesController#index as JSON
2022-01-20T07:14:55.084400101Z I, [2022-01-20T07:14:55.084293 #1]  INFO -- : [9b69831f-aac8-4d53-99f7-8063294233a8]   Parameters: {"account_id"=>"2"}
2022-01-20T07:14:55.084440114Z I, [2022-01-20T07:14:55.080042 #1]  INFO -- : [e219039c-5bac-46f4-ac79-36686d262c57] Started GET "/api/v1/accounts/2/custom_attribute_definitions" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.085491499Z I, [2022-01-20T07:14:55.085363 #1]  INFO -- : [e219039c-5bac-46f4-ac79-36686d262c57] Processing by Api::V1::Accounts::CustomAttributeDefinitionsController#index as JSON
2022-01-20T07:14:55.085632907Z I, [2022-01-20T07:14:55.085515 #1]  INFO -- : [e219039c-5bac-46f4-ac79-36686d262c57]   Parameters: {"account_id"=>"2"}
2022-01-20T07:14:55.087790156Z I, [2022-01-20T07:14:55.077254 #1]  INFO -- : [cd066f94-1299-4ace-8953-5ebc9e100f75] Processing by Api::V1::Accounts::TeamsController#index as JSON
2022-01-20T07:14:55.089982270Z I, [2022-01-20T07:14:55.089839 #1]  INFO -- : [cd066f94-1299-4ace-8953-5ebc9e100f75]   Parameters: {"account_id"=>"2"}
2022-01-20T07:14:55.097049569Z I, [2022-01-20T07:14:55.096903 #1]  INFO -- : [efebfd46-48b4-499b-808b-97f9536c84d3]   Rendered api/v1/accounts/labels/index.json.jbuilder (Duration: 2.5ms | Allocations: 697)
2022-01-20T07:14:55.102795015Z I, [2022-01-20T07:14:55.102662 #1]  INFO -- : [39edc88f-ee04-4a15-b4f1-6ff8d9683e34] Completed 200 OK in 20ms (Views: 0.1ms | ActiveRecord: 12.3ms | Allocations: 8246)
2022-01-20T07:14:55.103398948Z I, [2022-01-20T07:14:55.103273 #1]  INFO -- : source=rack-timeout id=39edc88f-ee04-4a15-b4f1-6ff8d9683e34 timeout=15000ms service=29ms state=completed
2022-01-20T07:14:55.109979303Z I, [2022-01-20T07:14:55.109852 #1]  INFO -- : source=rack-timeout id=92c4708c-aa50-4f6a-97d3-528f4387bb30 timeout=15000ms state=ready
2022-01-20T07:14:55.110809097Z I, [2022-01-20T07:14:55.110686 #1]  INFO -- : [efebfd46-48b4-499b-808b-97f9536c84d3] Completed 200 OK in 33ms (Views: 1.9ms | ActiveRecord: 24.2ms | Allocations: 13615)
2022-01-20T07:14:55.113268191Z I, [2022-01-20T07:14:55.113121 #1]  INFO -- : source=rack-timeout id=efebfd46-48b4-499b-808b-97f9536c84d3 timeout=15000ms service=44ms state=completed
2022-01-20T07:14:55.114368345Z I, [2022-01-20T07:14:55.114234 #1]  INFO -- : [92c4708c-aa50-4f6a-97d3-528f4387bb30] Started GET "/api/v1/accounts/2/conversations?status=open&assignee_type=me&page=1" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.115673801Z I, [2022-01-20T07:14:55.115536 #1]  INFO -- : [92c4708c-aa50-4f6a-97d3-528f4387bb30] Processing by Api::V1::Accounts::ConversationsController#index as JSON
2022-01-20T07:14:55.116043335Z I, [2022-01-20T07:14:55.115874 #1]  INFO -- : [92c4708c-aa50-4f6a-97d3-528f4387bb30]   Parameters: {"status"=>"open", "assignee_type"=>"me", "page"=>"1", "account_id"=>"2"}
2022-01-20T07:14:55.121392878Z I, [2022-01-20T07:14:55.119208 #1]  INFO -- : source=rack-timeout id=5276690c-c4cf-445d-8094-8ca358d523a2 timeout=15000ms state=ready
2022-01-20T07:14:55.122145683Z I, [2022-01-20T07:14:55.122026 #1]  INFO -- : [e219039c-5bac-46f4-ac79-36686d262c57]   Rendered api/v1/accounts/custom_attribute_definitions/index.json.jbuilder (Duration: 9.6ms | Allocations: 4394)
2022-01-20T07:14:55.123153093Z I, [2022-01-20T07:14:55.123022 #1]  INFO -- : [cd066f94-1299-4ace-8953-5ebc9e100f75]   Rendered api/v1/accounts/teams/index.json.jbuilder (Duration: 13.9ms | Allocations: 5886)
2022-01-20T07:14:55.125182208Z I, [2022-01-20T07:14:55.123384 #1]  INFO -- : [5276690c-c4cf-445d-8094-8ca358d523a2] Started GET "/api/v1/accounts/2/agents" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.129897477Z I, [2022-01-20T07:14:55.126341 #1]  INFO -- : [cd066f94-1299-4ace-8953-5ebc9e100f75] Completed 200 OK in 36ms (Views: 6.0ms | ActiveRecord: 25.1ms | Allocations: 13640)
2022-01-20T07:14:55.130644448Z I, [2022-01-20T07:14:55.130515 #1]  INFO -- : [5276690c-c4cf-445d-8094-8ca358d523a2] Processing by Api::V1::Accounts::AgentsController#index as JSON
2022-01-20T07:14:55.131279291Z I, [2022-01-20T07:14:55.131161 #1]  INFO -- : source=rack-timeout id=cd066f94-1299-4ace-8953-5ebc9e100f75 timeout=15000ms service=60ms state=completed
2022-01-20T07:14:55.132883007Z I, [2022-01-20T07:14:55.126812 #1]  INFO -- : [e219039c-5bac-46f4-ac79-36686d262c57] Completed 200 OK in 40ms (Views: 8.8ms | ActiveRecord: 22.6ms | Allocations: 15242)
2022-01-20T07:14:55.134518255Z I, [2022-01-20T07:14:55.134387 #1]  INFO -- : source=rack-timeout id=e219039c-5bac-46f4-ac79-36686d262c57 timeout=15000ms service=62ms state=completed
2022-01-20T07:14:55.135511755Z I, [2022-01-20T07:14:55.135404 #1]  INFO -- : source=rack-timeout id=2f6aa08e-3dde-44a6-92cf-2ded26eed4a8 timeout=15000ms state=ready
2022-01-20T07:14:55.139162651Z I, [2022-01-20T07:14:55.139030 #1]  INFO -- : [9b69831f-aac8-4d53-99f7-8063294233a8]   Rendered api/v1/accounts/inboxes/index.json.jbuilder (Duration: 20.6ms | Allocations: 7790)
2022-01-20T07:14:55.139937003Z I, [2022-01-20T07:14:55.139771 #1]  INFO -- : [9b69831f-aac8-4d53-99f7-8063294233a8] Completed 500 Internal Server Error in 54ms (ActiveRecord: 34.2ms | Allocations: 20535)
2022-01-20T07:14:55.142780993Z I, [2022-01-20T07:14:55.131408 #1]  INFO -- : [5276690c-c4cf-445d-8094-8ca358d523a2]   Parameters: {"account_id"=>"2"}
2022-01-20T07:14:55.145041626Z I, [2022-01-20T07:14:55.137640 #1]  INFO -- : source=rack-timeout id=1bd10eba-68a3-49da-b0af-0fd687305819 timeout=15000ms state=ready
2022-01-20T07:14:55.148318950Z F, [2022-01-20T07:14:55.148200 #1] FATAL -- : [9b69831f-aac8-4d53-99f7-8063294233a8]   
2022-01-20T07:14:55.148335606Z [9b69831f-aac8-4d53-99f7-8063294233a8] ActionView::Template::Error (Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true):
2022-01-20T07:14:55.148341470Z [9b69831f-aac8-4d53-99f7-8063294233a8]     1: json.id resource.id
2022-01-20T07:14:55.148346228Z [9b69831f-aac8-4d53-99f7-8063294233a8]     2: json.avatar_url resource.try(:avatar_url)
2022-01-20T07:14:55.148351226Z [9b69831f-aac8-4d53-99f7-8063294233a8]     3: json.channel_id resource.channel_id
2022-01-20T07:14:55.148355931Z [9b69831f-aac8-4d53-99f7-8063294233a8]     4: json.name resource.name
2022-01-20T07:14:55.148360432Z [9b69831f-aac8-4d53-99f7-8063294233a8]     5: json.channel_type resource.channel_type
2022-01-20T07:14:55.148364976Z [9b69831f-aac8-4d53-99f7-8063294233a8]   
2022-01-20T07:14:55.148369398Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/models/concerns/avatarable.rb:13:in `avatar_url'
2022-01-20T07:14:55.148373946Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/views/api/v1/models/_inbox.json.jbuilder:2
2022-01-20T07:14:55.148378299Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/views/api/v1/accounts/inboxes/index.json.jbuilder:3
2022-01-20T07:14:55.148382744Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/views/api/v1/accounts/inboxes/index.json.jbuilder:2
2022-01-20T07:14:55.148387119Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/views/api/v1/accounts/inboxes/index.json.jbuilder:1
2022-01-20T07:14:55.148391699Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/controllers/concerns/switch_locale.rb:24:in `set_locale'
2022-01-20T07:14:55.148396131Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/controllers/concerns/switch_locale.rb:16:in `switch_locale_using_account_locale'
2022-01-20T07:14:55.148400575Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/controllers/concerns/request_exception_handler.rb:11:in `handle_with_exception'
2022-01-20T07:14:55.148405028Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/controllers/concerns/switch_locale.rb:24:in `set_locale'
2022-01-20T07:14:55.148409529Z [9b69831f-aac8-4d53-99f7-8063294233a8] app/controllers/concerns/switch_locale.rb:11:in `switch_locale'
2022-01-20T07:14:55.148647970Z I, [2022-01-20T07:14:55.148524 #1]  INFO -- : source=rack-timeout id=9b69831f-aac8-4d53-99f7-8063294233a8 timeout=15000ms service=79ms state=completed
2022-01-20T07:14:55.149165122Z I, [2022-01-20T07:14:55.144050 #1]  INFO -- : [2f6aa08e-3dde-44a6-92cf-2ded26eed4a8] Started GET "/auth/validate_token" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.152289989Z I, [2022-01-20T07:14:55.149304 #1]  INFO -- : [1bd10eba-68a3-49da-b0af-0fd687305819] Started GET "/api/v1/accounts/2/" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.153097823Z I, [2022-01-20T07:14:55.152960 #1]  INFO -- : [2f6aa08e-3dde-44a6-92cf-2ded26eed4a8] Processing by DeviseOverrides::TokenValidationsController#validate_token as HTML
2022-01-20T07:14:55.157315164Z I, [2022-01-20T07:14:55.155071 #1]  INFO -- : [1bd10eba-68a3-49da-b0af-0fd687305819] Processing by Api::V1::AccountsController#show as JSON
2022-01-20T07:14:55.158033290Z I, [2022-01-20T07:14:55.157888 #1]  INFO -- : source=rack-timeout id=33c44246-1753-41ac-85ce-76b2f1653a02 timeout=15000ms state=ready
2022-01-20T07:14:55.162513577Z I, [2022-01-20T07:14:55.158446 #1]  INFO -- : [1bd10eba-68a3-49da-b0af-0fd687305819]   Parameters: {"id"=>"2"}
2022-01-20T07:14:55.164080778Z I, [2022-01-20T07:14:55.163944 #1]  INFO -- : [33c44246-1753-41ac-85ce-76b2f1653a02] Started GET "/cable" for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.169957126Z I, [2022-01-20T07:14:55.169820 #1]  INFO -- : [33c44246-1753-41ac-85ce-76b2f1653a02] Started GET "/cable/" [WebSocket] for 49.36.59.244 at 2022-01-20 07:14:55 +0000
2022-01-20T07:14:55.170999492Z I, [2022-01-20T07:14:55.170835 #1]  INFO -- : [33c44246-1753-41ac-85ce-76b2f1653a02] Successfully upgraded to WebSocket (REQUEST_METHOD: GET, HTTP_CONNECTION: upgrade, HTTP_UPGRADE: websocket)
2022-01-20T07:14:55.171374108Z I, [2022-01-20T07:14:55.171242 #1]  INFO -- : source=rack-timeout id=33c44246-1753-41ac-85ce-76b2f1653a02 timeout=15000ms service=13ms state=completed
2022-01-20T07:14:55.181004346Z I, [2022-01-20T07:14:55.180865 #1]  INFO -- : [92c4708c-aa50-4f6a-97d3-528f4387bb30]   Rendered api/v1/accounts/conversations/index.json.jbuilder (Duration: 3.9ms | Allocations: 1320)
2022-01-20T07:14:55.185057774Z I, [2022-01-20T07:14:55.184919 #1]  INFO -- : [92c4708c-aa50-4f6a-97d3-528f4387bb30] Completed 200 OK in 66ms (Views: 18.7ms | ActiveRecord: 25.2ms | Allocations: 25952)
2022-01-20T07:14:55.185622787Z I, [2022-01-20T07:14:55.185502 #1]  INFO -- : source=rack-timeout id=92c4708c-aa50-4f6a-97d3-528f4387bb30 timeout=15000ms service=76ms state=completed
2022-01-20T07:14:55.196076318Z I, [2022-01-20T07:14:55.195944 #1]  INFO -- : [1bd10eba-68a3-49da-b0af-0fd687305819]   Rendered api/v1/accounts/show.json.jbuilder (Duration: 0.9ms | Allocations: 331)
2022-01-20T07:14:55.199131393Z I, [2022-01-20T07:14:55.198966 #1]  INFO -- : [1bd10eba-68a3-49da-b0af-0fd687305819] Completed 200 OK in 36ms (Views: 1.6ms | ActiveRecord: 17.4ms | Allocations: 14398)
2022-01-20T07:14:55.199570077Z I, [2022-01-20T07:14:55.199444 #1]  INFO -- : source=rack-timeout id=1bd10eba-68a3-49da-b0af-0fd687305819 timeout=15000ms service=62ms state=completed
2022-01-20T07:14:55.202587531Z I, [2022-01-20T07:14:55.202457 #1]  INFO -- : [2f6aa08e-3dde-44a6-92cf-2ded26eed4a8]   Rendered devise/token.json.jbuilder (Duration: 36.7ms | Allocations: 13146)
2022-01-20T07:14:55.204958078Z I, [2022-01-20T07:14:55.204820 #1]  INFO -- : [5276690c-c4cf-445d-8094-8ca358d523a2]   Rendered api/v1/accounts/agents/index.json.jbuilder (Duration: 44.5ms | Allocations: 17413)
2022-01-20T07:14:55.206141712Z I, [2022-01-20T07:14:55.206008 #1]  INFO -- : [2f6aa08e-3dde-44a6-92cf-2ded26eed4a8] Completed 200 OK in 51ms (Views: 24.2ms | ActiveRecord: 21.5ms | Allocations: 19432)
2022-01-20T07:14:55.206653094Z I, [2022-01-20T07:14:55.206525 #1]  INFO -- : source=rack-timeout id=2f6aa08e-3dde-44a6-92cf-2ded26eed4a8 timeout=15000ms service=71ms state=completed
2022-01-20T07:14:55.207286145Z I, [2022-01-20T07:14:55.206951 #1]  INFO -- : [5276690c-c4cf-445d-8094-8ca358d523a2] Completed 200 OK in 63ms (Views: 21.7ms | ActiveRecord: 33.8ms | Allocations: 25021)
2022-01-20T07:14:55.208107859Z I, [2022-01-20T07:14:55.207992 #1]  INFO -- : source=rack-timeout id=5276690c-c4cf-445d-8094-8ca358d523a2 timeout=15000ms service=89ms state=completed
2022-01-20T07:14:55.224312697Z I, [2022-01-20T07:14:55.224146 #1]  INFO -- : RoomChannel is streaming from ArzPczNxJyZSkTPGeeUDt7WG
2022-01-20T07:15:00.422915394Z I, [2022-01-20T07:15:00.422750 #1]  INFO -- : Finished "/cable/" [WebSocket] for 49.36.59.244 at 2022-01-20 07:15:00 +0000
2022-01-20T07:15:00.423053255Z I, [2022-01-20T07:15:00.422936 #1]  INFO -- : RoomChannel stopped streaming from ArzPczNxJyZSkTPGeeUDt7WG
2022-01-20T07:15:08.712502610Z I, [2022-01-20T07:15:08.712285 #1]  INFO -- : RoomChannel#update_presence
2022-01-20T07:15:28.710872959Z I, [2022-01-20T07:15:28.710649 #1]  INFO -- : RoomChannel#update_presence
2022-01-20T07:15:48.718364147Z I, [2022-01-20T07:15:48.718141 #1]  INFO -- : RoomChannel#update_presence
2022-01-20T07:16:08.717685659Z I, [2022-01-20T07:16:08.717494 #1]  INFO -- : RoomChannel#update_presence
2022-01-20T07:16:28.711316719Z I, [2022-01-20T07:16:28.711048 #1]  INFO -- : RoomChannel#update_presence
2022-01-20T07:16:48.714172403Z I, [2022-01-20T07:16:48.713934 #1]  INFO -- : RoomChannel#update_presence
2022-01-20T07:17:08.717688023Z I, [2022-01-20T07:17:08.717463 #1]  INFO -- : RoomChannel#update_presence
2022-01-20T07:17:28.715368294Z I, [2022-01-20T07:17:28.715149 #1]  INFO -- : RoomChannel#update_presence

**Comment by tejaswinichile** [comment]
Please check   Rails.application.routes.default_url_options is set properly, according to the logs avatar_url is not able to build the URL fetch the profile image which has been uploaded.

The issue is with the env config not related to uploads.

**Comment by vedangdate** [comment]
Is it possible to do this through the portal?
I checked both CapRover and Chatwoot, but I couldn't see such a setting on the portals.
Will I have to manually change the configuration in the code?
I see that this setting is there in a file called "production.rb"

**Comment by tejaswinichile** [comment]
You will have to change the .env file and restart the server again with 
```
# Replace with the URL you are planning to use for your app
FRONTEND_URL=http://0.0.0.0:3000
```

**Comment by vedangdate** [comment]
So I did try just adding the variable FRONTEND_URL in the CapRover's env variables section, just to see if my hunch would work and the issue was resolved. *cheers silently
I didn't need to make any changes in the actual code so am guessing the .env file has all it's properties in the CapRover's web-portal appConfig section.
Thank you for pointing out the issue! That saved me a lot of time. @tejaswinichile
